ui/shared/components/page/AcceptPolicies.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Cookies from 'js-cookie'
+import { Modal } from 'semantic-ui-react'
+import { connect } from 'react-redux'
+import { Route, Switch, Link } from 'react-router-dom'
+
+import { updateUserPolicies } from 'redux/rootReducer'
+import { getUser } from 'redux/selectors'
+import { BooleanCheckbox } from '../form/Inputs'
+import ReduxFormWrapper, { validators } from '../form/ReduxFormWrapper'
+
+
+const COOKIE_ACTIONS = [{
+  primary: true,
+  content: 'Accept',
+  onClick: () => {
+    Cookies.set('accepted_cookies', true)
+    window.location.href = window.location.href
+  },
+}]
+
+const AcceptCookies = () =>
+  Cookies.get('accepted_cookies') ||
+  <Modal
+    open
+    size="small"
+    closeOnDimmerClick={false}
+    closeOnEscape={false}
+    header="This website uses cookies"
+    content={
+      <Modal.Content>
+        seqr collects cookies to improve our user experience and ensure the secure functioning of our site. For more
+        details, see our <Link target="_blank" to="/privacy_policy">Privacy Policy</Link>. By
+        clicking &quot;Accept&quot;, you consent to the use of these cookies.
+      </Modal.Content>
+    }
+    actions={COOKIE_ACTIONS}
+  />
+
+
+const POLICY_FORM_FIELDS = [
+  {
+    name: 'acceptedPolicies',
+    component: BooleanCheckbox,
+    validate: validators.required,
+    label: (
+      // eslint-disable-next-line jsx-a11y/label-has-for
+      <label>
+        I accept the <Link target="_blank" to="/terms_of_service">Terms of Service</Link> and
+        the <Link target="_blank" to="/privacy_policy">Privacy Policy</Link>
+      </label>
+    ),
+  },
+]
+
+export const AcceptPolicies = React.memo(({ user, onSubmit }) => (
```

Comment:
There should not be an `export` here.

Reply:
this is needed for testing, I will change this to `BaseAcceptPolicies` and add a comment to make it clearer